spyder/config/main.py

```diff
@@ -494,8 +499,90 @@
               'advanced/port': 2087,
               'advanced/external': False,
               'advanced/stdio': False
-             })
+             }),
+            ]
+
+
+NAMEMAP = {

```

Comment:
This is how config folder looks now @ccordoba12 

<img width="212" alt="Screen Shot 2019-07-17 at 00 40 41" src="https://user-images.githubusercontent.com/3627835/61349950-8b8f5f80-a82b-11e9-98a7-bad2b75d572e.png">

I added the lsp just to test, but we need to define what file names we want and what options to include

Reply:
Let's rename this constant to `NAME_MAP`